components/vocabularies/ui/src/main/resources/PhenoTips/IndexVocabulary.xml

```diff
@@ -428,10 +406,8 @@ $xwiki.ssx.use('PhenoTips.Widgets')##
         // Create a notification message to display to the user when the submit is being sent
         form._x_notification = new XWiki.widgets.Notification("$!escapetool.javascript($services.localization.render('phenotips.indexVocabulary.inProgress'))", "inprogress");
         form.disable();
-        // FIXME Use the XWiki.Document JS API after upgrading to XWiki 6.0
-        new Ajax.Request("$xwiki.getURL('PhenoTips.IndexVocabulary', 'get')", {
-            method : 'post',
-            parameters : formData,
+        var vocabularyIndexURL = XWiki.contextPath + "/rest/vocabularies/" + vocabularyId + "?method=POST&amp;url=" + formData._object.url;
```

Comment:
Why `formData._object.url`? `_object` is not a public API. Above `formData` is created as a [`Hash`](http://api.prototypejs.org/language/Hash/), which has a [`get` method](http://api.prototypejs.org/language/Hash/prototype/get/). You could also just append `formData.toQueryString()`, in case we'll ever need more than just a simple url.

There's also some code above that can be removed, setting extra parameters in `formData` that are now unused.